### PR TITLE
Config for JitPack.io

### PR DESCRIPTION
**Pull request by ipcjs**
Make the project can be automatically compiled on [jitpack.io](https://jitpack.io/#googlemaps/android-maps-utils) by judging whether there is [a JITPACK variable](https://jitpack.io/docs/BUILDING/#build-environment).

### CODE REVIEW

**Comment by stephenmcd** [comment]
@domesticmouse any thoughts?

**Comment by stephenmcd** [comment]
@ipcjs can you describe what this achieves?

File: library/build.gradle [hunk 1]
```diff
@@ -35,106 +30,15 @@ android {
...
-
-        developers {
-            developer {
-                id 'broady'
-                name 'Chris Broadfoot'
-                url 'http://google.com/+ChristopherBroadfoot'
-            }
-        }
-    }
+if (System.getenv("JITPACK")) {
```
Comment (ipcjs) [inline library/build.gradle]: see: https://jitpack.io/docs/BUILDING/#build-environment

**Comment by ipcjs** [comment]
@stephenmcd I updated this description.

**Comment by barbeau** [comment]
My understanding is that JitPack serves as a "maven central" for projects compiled directly from GitHub commits/releases.

@ipcjs in this projects case, I think the main benefit of enabling JitPack is that it would allow users of the library to use any branch or commit as their dependency. So users could use the current master branch head commit as a direct dependency for their project from JitPack, without having to depend on a release or build the project locally themselves. Is this right? Are there other benefits?

**Comment by ipcjs** [comment]
@barbeau Yes, you are right, that is all.

**Comment by stephenmcd** [comment]
Ok that seems very worthwhile - thanks all for the input.

**Pull request merged**